### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by vanko6226**

### CODE REVIEW

File: README.md
Comment (NikOneZ1) [inline README.md:1]: Why have you changed this file?

File: app/players/elves/elf_ranger.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str, musical_instrument: str,
+                 bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return (f"elves ranger {self.nickname}."
```
Comment (NikOneZ1) [inline app/players/elves/elf_ranger.py:11]: Returned text is different from the text in task
```suggestion
        return (f"Elf ranger {self.nickname}."
```

File: tests/test_main.py
Comment (NikOneZ1) [inline tests/test_main.py:1]: Why have you changed this file?

**Review by NikOneZ1 — CHANGES_REQUESTED** [review]

File: README.md
Comment (vanko6226) [reply README.md:1]: I myself don’t understand why this file has changed. I will sort this out

File: tests/test_main.py
Comment (vanko6226) [reply tests/test_main.py:1]: I myself don’t understand why this file has changed. I will sort this out

**Pull request closed without merging**